runPerftool.psm1

```diff
@@ -332,55 +358,58 @@ Function ProcessToolCommands{
         )
         [bool] $gracefulCleanup = $False
         # delay to let credential (public key) propagate before remoting
-        $credPropagationTimeInSecond = 2
+        $credPropagationTimeInSecond = 3
     
         [System.IO.TextReader] $recvCommands = $null
         [System.IO.TextReader] $sendCommands = $null
     
         $toolpath = "./{0}" -f $Toolname
-    
         $homePath = "/home/$TestUserName"
```

Comment:
Should we just use the ~ shortcut here? Do we always assume the passed in user name is the same as the running user?

Reply:
When running on elevated privileges with Invoke-AzRunCommand, ~ does not exist.